game-core/src/test/java/games/strategy/triplea/ai/pro/util/ProTerritoryValueUtilsTest.java

```diff
@@ -62,4 +64,14 @@ void testFindNearbyEnemyCapitalsAndFactoriesNoneWithinDistance9() {
         result,
         containsInAnyOrder(northJapan));
   }
+
+  @Test
```

Comment:
Good to see the test case, can we increase the number of tests and thoroughness? Are the edge cases tested?

Reply:
@asvitkine thoughts on if there are additional test cases to make sure the testing is thorough?
Perhaps the not found case, or the find at the zero distance, or cases where we should find but it's past max distance?

Reply:
findMaxLandMassSize() doesn't really have parameters, so the only way to try different cases is with different maps.

I added cases for Revised and Minimap (which is a single small continent). For the latter, I added its XML file to test data. Let me know if you think this is OK.